[color-picker] First stab at integrating `<space-picker>` (#155)
color-js/elements: src/color-picker/README.md

</color-picker>
```

> or the default space picker:

Reviewer: ```suggestion
or your own form element instead of the default space picker:
```

```html
<color-picker space="oklch" color="oklch(60% 30% 180)">

> 	<select slot="space-picker" size="3" class="horizontal">

Author: Agreed. I'll fix it

		<option value="oklch" selected>OKLCh</option>
		<option value="hwb">HWB</option>
		<option value="hpluv">HPLuv</option>
	</select>
</color-picker>

<style>
	select.horizontal {
		writing-mode: tb;

> 		field-sizing: content;

Reviewer: The horizontal select trick is nice, but not a good option for a demo, you want anything that is not relevant to the demo itself to be relatable, not to have people scratching their heads

Author: Got it. I'll rework it. Thanks!

</color-picker>
```

> <!-- ### Dynamic

Reviewer: This is still useful! But we may want to rework it a bit so that it's not literally replicating a space picker.

Author: Yeah, I just commented out so we could rework it.

Author: I still don't know what the best alternative for this is. Moreover, I have already demonstrated something similar in the Slots section. I'd be grateful for any ideas.

| Name | Description |
|------|-------------|
| (default) | The color picker's main content. Goes into the swatch. |

> | `color-space` | An element used to set the picker color space. |

Reviewer: ```suggestion
| `color-space` | An element to display (and if writable, also set) the current color space. If not provided, a `<space-picker>` is used. |
```

(with `<space-picker>` being a link)

| `space` | `space` | `ColorSpace` &#124; `string` | `oklch` | The color space to use for interpolation. |
| `color` | `color` | `Color` &#124; `string` | `oklch(50% 50% 180)` | The current color value. |

### Getters

These properties are read-only.

| Property | Type | Description |
|----------|------|-------------|

> | `spaceResolved` | `ColorSpace` | Color space object corresponding to the `space` property. |

Reviewer: Given that in JS, `spaceResolved` is far more useful than `space`, I wonder if we should make `space` the one with the actual color space (and make it read/write), and `spaceRaw` or something the one that reflects. Is there any precedent in HTML where the object prop is the one with the "nicer" name?

### Events

| Name | Description |
|------|-------------|

> | `input` | Fired when the color changes due to user action, either with the space picker, sliders or the color swatch's input field. |

Reviewer: ```suggestion
| `input` | Fired when the color changes due to user action, such as adjusting the sliders, entering a color in the swatch's text field, or choosing a different color space. |
```

(and same for the one below)
